etc/kayobe/kolla.yml

```diff
@@ -151,6 +151,8 @@ kolla_build_customizations:
    # Use our own repo server to install the monasca-grafana fork
    base_yum_repo_files_remove:
      - 'grafana.repo'
+   influxdb_packages_override:
+     - 'influxdb-0.10.3-1'
```

Comment:
The package comes from this repo which is configured by default in the base image: https://repos.influxdata.com/rhel/7/x86_64/stable/